cmd/main.go

```diff
@@ -164,18 +169,30 @@ func main() {
 	signal.Notify(c, os.Interrupt)
 
 	// subscribing to nats
-	natsClient, err := nats.New("<server-url>")
+	natsClient, err := nats.New("nats://127.0.0.1:4222")
 	if err != nil {
 		logrus.Printf("Nats client create error %v", err)
 	} else {
-		err = natsClient.Subscribe("cluster")
+		msgch := make(chan *broker.Message)
+		err = natsClient.SubscribeWithChannel("cluster", "cluster-queue", msgch)
 		if err != nil {
 			logrus.Printf("Error subscribing to cluster %v", err)
 		}
-		err = natsClient.Subscribe("istio")
+		err = natsClient.SubscribeWithChannel("istio", "istio-queue", msgch)
```

Comment:
This approach is primarily for purposes of getting initial end-to-end functionality going, I assume, with the intent to refactor by including subscriptions based on actively connected Meshery adapters.

Reply:
Yes yes, for the initial end to end.